### PR TITLE
Adds ARG assault rifle to the game.

### PR DESCRIPTION
**Pull request by GoonOnMoon**
Adds ARG assault rifle to the list of weapons spawnable. Only available through badminnery as of now.

Added myself as a coder to admins.config.

Added a sprite for the ARG, inhands and the item.

Gun was a bit bugged, took two commits to fix it. Works nicely now.

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 1]
```diff
@@ -246,4 +240,17 @@
 	fire_sound = 'sound/weapons/Gunshot_smg.ogg'
 	can_suppress = 0
 	burst_size = 4
+	fire_delay = 1
+
+/obj/item/weapon/gun/projectile/automatic/ar
```
Comment (Alek2ander) [inline code/modules/projectiles/guns/projectile/automatic.dm]: Arrrrrrrr

Comment (GoonOnMoon) [inline code/modules/projectiles/guns/projectile/automatic.dm]: AR as in assault rifle :)

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 2]
```diff
@@ -246,4 +240,17 @@
...
 	burst_size = 4
+	fire_delay = 1
+
+/obj/item/weapon/gun/projectile/automatic/ar
+	name = "arg"
+	desc = "A robust assault rile used by Nanotrasen fighting forces."
+	icon_state = "arg"
+	item_state = "arg"
+	slot_flags = 0
+	origin_tech = "combat=5;materials=1;syndicate=2"
```
Comment (GunHog) [inline code/modules/projectiles/guns/projectile/automatic.dm]: >A robust assault rile used by <b>Nanotrasen</b> fighting forces.

Then why does it have a Syndicate tech origin?

**Comment by GunHog** [comment]
Why is it admin only? It is 35 dmg bullet, right? Why not add it to Summon Guns, at least? It is such a waste of content to put an item in but not any way of spawning it.

**Comment by SconesC** [comment]
You should post screenshots

**Comment by AlexanderUlanH** [comment]
:-1: NT = Energy weapons; Syndie = Ballistic weapons, and there's no shortage of NT weapons that are in line with that split.  If this had a real reason to exist, I wouldn't mind, but there's no sense adding something for thematic purposes that goes against the established theme.

**Comment by GoonOnMoon** [comment]
35dmg bullet with a 3round burst. I'll add it to summon guns in my next PR!

File: code/modules/projectiles/guns/projectile/automatic.dm (hunk 2 above)
Comment (GoonOnMoon) [inline code/modules/projectiles/guns/projectile/automatic.dm]: Changed this in my second commit.

**Comment by GoonOnMoon** [comment]
http://imgur.com/c0onYox

Heard y'all want some SCREENIES

**Comment by GoonOnMoon** [comment]
Going to close this PR and change some things!

File: code/modules/projectiles/guns/projectile/automatic.dm (hunk 1 above)
Comment (Miauw62) [inline code/modules/projectiles/guns/projectile/automatic.dm]: ar is a pretty shitty type name, use assault or something.

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 3]
```diff
@@ -246,4 +240,17 @@
 	fire_sound = 'sound/weapons/Gunshot_smg.ogg'
 	can_suppress = 0
 	burst_size = 4
+	fire_delay = 1
+
+/obj/item/weapon/gun/projectile/automatic/ar
+	name = "arg"
```
Comment (Miauw62) [inline code/modules/projectiles/guns/projectile/automatic.dm]: arg is a pretty shitty name, not really obvious that it's a gun. try making it capital letters or something (if the first letter is a capital letter, add /improper in front of the name)

**Comment by Miauw62** [comment]
also why is this even needed.
as others have said, NT is generally energy weapons while the syndicate is more of a ballistic thing. we already have quite a few really powerful guns, this gun doesn't have anything special and isn't really needed.

**Pull request closed without merging**